docs/sphinx/using/cudaq/platform.rst

```diff
@@ -104,32 +117,161 @@ should be used.
-    :end-before: [End Documentation]
+    .. literalinclude:: ../../snippets/cpp/using/cudaq/platform/observe_mqpu_mpi.cpp
+        :language: cpp
+        :start-after: [Begin Documentation]
+        :end-before: [End Documentation]
 
-.. code-block:: console
+    .. code-block:: console
 
-    nvq++ file.cpp -target nvidia-mqpu
-    mpirun -np <N> ./a.out
+        nvq++ file.cpp -target nvidia-mqpu
+        mpiexec -np <N> a.out
 
 
-Python
-^^^^^^
+.. tab:: Python
 
-.. literalinclude:: ../../snippets/python/using/cudaq/platform/observe_mqpu_mpi.py
-    :language: python
-    :start-after: [Begin Documentation]
+    .. literalinclude:: ../../snippets/python/using/cudaq/platform/observe_mqpu_mpi.py
+        :language: python
+        :start-after: [Begin Documentation]
 
-.. code-block:: console
+    .. code-block:: console
 
-    mpirun -np <N> python3 file.py
+        mpiexec -np <N> python3 file.py
 
 In the above example, the parallel distribution mode was set to :code:`mpi` using :code:`cudaq::parallel::mpi` in C++ or :code:`cudaq.parallel.mpi` in Python.
 CUDA Quantum provides MPI utility functions to initialize, finalize, or query (rank, size, etc.) the MPI runtime. 
 Last but not least, the compiled executable (C++) or Python script needs to be launched with an appropriate MPI command, 
-e.g., :code:`mpirun`, :code:`mpiexec`, :code:`srun`, etc.
+e.g., :code:`mpiexec`, :code:`mpirun`, :code:`srun`, etc.
+
+.. _remote-mqpu-platform:
+
+Remote `MQPU` Platform
++++++++++++++++++++++++++++
+
+As shown in the above examples, the :code:`nvidia-mqpu` platform enables
+multi-QPU distribution whereby each QPU is simulated by a :ref:`single NVIDIA GPU <cuQuantum single-GPU>`.
+
+To run multi-QPU workloads on different simulator backends, one can use the :code:`remote-mqpu` platform,
+which encapsulates simulated QPUs as independent HTTP REST server instances. 
+Here is an example.
+
+.. tab:: C++
+
+    .. literalinclude:: ../../snippets/cpp/using/cudaq/platform/sample_async_remote.cpp
+        :language: cpp
+        :start-after: [Begin Documentation]
+        :end-before: [End Documentation]
+
+    .. code-block:: console
+
+        nvq++ sample_async.cpp -o sample_async.x --target remote-mqpu --remote-mqpu-auto-launch 2
+        ./sample_async.x
+
+
+.. tab:: Python
+
+    .. literalinclude:: ../../snippets/python/using/cudaq/platform/sample_async_remote.py
+        :language: python
+        :start-after: [Begin Documentation]
+
+In the above code snippets, the :code:`remote-mqpu` platform was used in the auto-launch mode,
+whereby a specific number of server instances, i.e., virtual QPUs, are launched on the local machine
+in the background. The remote QPU daemon service, :code:`cudaq-qpud`, will also be shut down automatically
+at the end of the session.
+
+.. note:: 
+    By default, auto launching daemon services do not support MPI parallelism. 
+    Hence, multi-GPU backend capabilities are not available with auto-launch. 
```

Comment:
I would rephrase this note to:
```
By default, auto launching daemon services do not support MPI parallelism.
Hence, using the `nvidia-mgpu` backend to simulate each virtual QPU requires manually launching each server instance. How to do that is explained in the rest of this section.
```

Reply:
Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1012/commits/20de9c01271dd3c8455113ef1d1dce95e116cead